lib/net.js

```diff
@@ -991,10 +990,7 @@ Socket.prototype.connect = function(...args) {
 
 
 function lookupAndConnect(self, options) {
-  var host = options.host || 'localhost';
-  var port = options.port;
-  var localAddress = options.localAddress;
-  var localPort = options.localPort;
+  var { host = 'localhost', port, localAddress, localPort } = options;
```

Comment:
This is not exactly the same as before. I think it fine but `host = 'localhost` !== options.host || 'localhost'`. The first part is only going to set it to localhost in case host is undefined while the latter is going to do that for any falsy value.

Reply:
It's not strict equal but doesn't break CI. Should I revert it?

Reply:
I would prefer it if we reverted this change, or at least the part with `localhost`, other changes LGTM